<launch>
    <arg name="bag" />
    <node pkg="hrl_phri_2011" type="pc_head_stitcher" name="pc_head_stitcher" output="screen"
          args="$(arg bag)" >
        <param name="icp_radius" value="0.03" />
        <param name="icp_iters" value="30" />
        <param name="use_rgb" value="1" />
        <param name="color_weight" value="0.0008" />
        <param name="z_cutoff" value="-0.2" />
        <param name="keep_radius" value="0.001" />
        <param name="num_erode" value="2" />

        <!--
        <param name="icp_radius" value="0.03" />
        <param name="icp_iters" value="100" />
        <param name="use_rgb" value="1" />
        <param name="color_weight" value="0.0008" />
        <param name="z_cutoff" value="-0.2" />
        <param name="keep_radius" value="0.001" />
        <param name="num_erode" value="2" />
        -->
    </node>
</launch>

